packages/typespec-vscode/src/vscode-cmd/create-tsp-project.ts

```diff
@@ -195,71 +159,131 @@ export async function createTypeSpecProject(
         return;
       }
 
+      const defaultLibraries =
+        info.template.libraries?.map((lib) => {
+          if (typeof lib === "string") {
+            return { name: lib };
+          }
+          return lib;
+        }) ?? [];
       const initTemplateConfig: InitProjectConfig = {
         template: info.template!,
         directory: selectedRootFolder,
-
         baseUri: info.baseUrl,
         name: projectName!,
         parameters: inputs ?? {},
-        includeGitignore: includeGitignore,
-        libraries: librariesToInclude,
+        includeGitignore: true,
```

Comment:
same here, just keep the default which is already `true`

Reply:
could you help to double confirm? i think these default values are not handled by ```scaffoldNewProject()``` directly but  set by ```makeScaffoldingConfig()``` in ```tsp init``` logic before calling ```scaffoldNewProject()```.